### DISCUSSION TITLE
Lsp in jupyter

### DISCUSSION THREAD

**Discussion by russarr** [glacambre/firenvim#1574] [Q&A]
I have a question.
I want to use firenvim to work in jupyter notebook. I have lsp(pyright) configured in nvim. But unfortunately its scope is only the current cell.
For example, if I define a variable in the previous cell. And then I open the next cell in firenvim for editing, then pyright does not see this variable.
Is there any way to have a common namespace for all jupyter cells from one notebook?

**Comment by glacambre** [comment]
Hi, thanks for using Firenvim! Unfortunately, at the moment, there's no way to do this.
